### ISSUE TITLE
zeppelin prepare

### ISSUE THREAD

**Issue by wdavidw** [ryba-io/ryba#50]
build the tar.gz from git and run it inside a docker container

**Comment by IDerr** [comment]
Any news on this @wdavidw

**Comment by wdavidw** [comment]
No, not a priority on my side, hope @lucasbak will have more time

**Comment by lucasbak** [comment]
Yes it's on the roadmap , but I was thinking about rewriting  zeppelin to use GDP package directly

**Comment by wdavidw** [comment]
@lucasbak cant we remove the solr embeded support in ranger ?

**Comment by lucasbak** [comment]
We can its was just développés to make use of ryba cluster more easy
